summarization/html_parsers/index_parser.py

```diff
@@ -52,8 +44,17 @@ def get_article_text(self, url, soup) -> str:
         # remove lead if exists
         if article:
             leads = article.select('div.cikk-torzs > p > strong')
+            tags = article.select('div.cikk-torzs > div > ul.m-tag-list')
+            asides = article.select('div.cikk-torzs > aside')
+            content_disclaimer = soup.select('div.cikk-torzs > div.content-disclaimer-text')
             for lead in leads:
                 lead.decompose()
+            for tag in tags:
+                tag.decompose()
+            for aside in asides:
+                aside.decompose()
+            for element in content_disclaimer:
+                element.decompose()
```

Comment:
You can collect every tag you want to remove in one list, and then one for loop is enough.